app/bundles/EmailBundle/Swiftmailer/Momentum/Facade/MomentumFacadeInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Mautic\EmailBundle\Swiftmailer\Momentum\Facade;
+
+use Mautic\EmailBundle\Swiftmailer\SwiftmailerFacadeInterface;
+
+/**
+ * Interface MomentumApiFacadeInterface.
```

Comment:
Nope, but why do you need this empty interface which extends another interface anyway?